terraform/modules/monitoring/grafana.tf

```diff
@@ -4,7 +4,8 @@ locals {
 
 module "grafana_app" {
   source                        = "../app"
-  image_name                    = "grafana/grafana"
+  image_name                    = "grafana"
+  registry_fqdn                 = "grafana"
```

Comment:
If specifying a short name works (which I presume it does) then perhaps `registry_fqdn` isn't the best name. Maybe just `registry`?

Reply:
yeah, a short name works. I can fix that name to `registry`. I forgot about the Grafana part until the last minute.